plugins/org.eclipse.reddeer.eclipse/src/org/eclipse/reddeer/eclipse/ui/console/ConsoleView.java

```diff
@@ -244,4 +245,22 @@ public String getConsoleLabel(){
 		org.eclipse.swt.widgets.Widget swtWidget = widgetIsFound.getResult();
 		return (swtWidget == null) ? null : LabelHandler.getInstance().getText((org.eclipse.swt.widgets.Label)swtWidget);
 	}
+
+	/**
+	 * Returns a control registered via adapters. This is usually StyledText or
+	 * Canvas.
+	 * 
+	 * @return registered control
+	 */
+	protected Control getRegisteredControl() {
+		activate();
+		WidgetIsFound widgetIsFound = new WidgetIsFound(org.eclipse.swt.custom.StyledText.class, cTabItem.getControl());
+		new WaitUntil(widgetIsFound, TimePeriod.SHORT, false);
+		// Check whether there is a console to display or not
+		if (widgetIsFound.getResult() == null) {
+			return null;
```

Comment:
Please add debug message which will inform user  that console view does not have any console